src/Web/Twitter/Feed.hs

```diff
@@ -43,13 +44,17 @@ timeline oauth credential count excludeReplies username = do
       Right ts     -> Right $ map (simplifyTweet . linkifyTweet) ts
 
 createRequest :: String -> Int -> Bool -> IO Request
-createRequest username count excludeReplies = parseUrl $ timelineUrl username count excludeReplies
+createRequest username count excludeReplies = parseUrlThrow $ timelineUrl username count excludeReplies
+#if (!MIN_VERSION_http_conduit(2,1,11))
+  where
+    parseUrlThrow = parseUrl
+#endif
 
 getResponse :: OAuth -> Credential -> Request -> IO (Response BS.ByteString)
-getResponse oauth credential req =
-  withManager $ \m -> do
-    signedreq <- signOAuth oauth credential req
-    httpLbs signedreq m
+getResponse oauth credential req = do
+  m <- newManager tlsManagerSettings
```

Comment:
@CristhianMotoche We can do `defaultManagerSettings` instead to keep it simpler. We can add the import:

```haskell
import Network.HTTP.Client.Conduit (defaultManagerSettings)
```

And use that one instead of `tlsManagerSettings`. This way we don't need the http-client-tls dependency and we don't get the warning for redundant imports for some versions.

Let me know if you can make this change. I can make it locally and merge otherwise.